spec/models/event_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+describe "Event model", type: :model do
+
+    it "should not validate when everything is specified correctly" do
+        valid = League.new(name: "Weitwurfmeisterschaften",
```

Comment:
Isn't this a typical case of using FactoryBot instead of hardcoding the values? If you want a certain value to not be defined just set it to nil.